doc/source/configuration/wazuh.rst

```diff
@@ -260,8 +260,8 @@ By default, Wazuh Ansible uses `wazuh-cert-tool.sh <https://documentation.wazuh.
 to automatically
 generate certificates for wazuh-indexer (previously Elasticsearch and opendistro)
 and wazuh-dashbooard (previously Kibana) using a local CA.
-If the certificates directory ``etc/kayobe/ansible/wazuh/certificates``
-does not exist, it will generate the following certificates in ``etc/kayobe/ansible/wazuh/certificates/certs/``
+If the certificates directory ``{{ kayobe_env_path }}/wazuh-certificates``
```

Comment:
Should this be `kayobe_env_config_path`?

Reply:
Also wazuh/certificates, not wazuh-certificates

Reply:
done